workbench/test/selenium_test.py

```diff
@@ -10,7 +9,6 @@
 from workbench.runtime_util import reset_global_state
 
 
-@attr('selenium')
```

Comment:
Do you want to change this to a pytest mark instead?  (pytest can use the nose attrib pligin attrs also, but I'm all for gradually ripping out nose usage.)

Reply:
I wasn't sure we were actually using them for anything, but I guess it makes sense to keep them around for running those tests in isolation. I'll do that.